### DISCUSSION TITLE
Contributing instructions - tests fail

### DISCUSSION THREAD

**Discussion by nfplee** [nhibernate/nhibernate-core#3599] [General] [answered]
I followed the instructions from the [CONTRIBUTING.md](https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md) file and everything was fine until I tried running the tests. I've tried running both the C and D options from the ShowBuildMenu.bat menu, but in both cases, many tests fail.

FYI, I'm testing using SQL Server 2008. Here's the TestResult.xml (renamed to .txt so that I could upload it):

[TestResult.txt](https://github.com/user-attachments/files/16714283/TestResult.txt)

I'd appreciate any help, as I'd like to submit a pull request for an issue I'm experiencing, but I want to make sure my pull request doesn't break anything.

**Comment by fredericDelaporte** [comment, marked as answer]
Search test results with label "error" (searching `"error"` is enough):

> Could not open connection to: Data Source=tdb2008r2

Occurs on tests requiring at least two concurrent connections: is your db somehow in single user mode?

> Network access for Distributed Transaction Manager (MSDTC) has been disabled. Please enable DTC for network access in the security configuration for MSDTC using the Component Services Administrative tool.

Many tests test distributed transactions, so DTC has to be enabled. That is a system thing.

**Comment by nfplee** [comment]
Thanks, I installed a local instance of SQL Server 2019 Express for testing and it came back with no failures, but 76 are inconclusive and skipped 714. Here's the TestResult.xml:

[TestResult.txt](https://github.com/user-attachments/files/16726318/TestResult.txt)

Please let me know if that's good enough to go on and then I'll submit my pull request (providing I get no failures after the fix is applied).

**Reply by fredericDelaporte** [reply]
Skipped and inconclusive tests are expected. We test many vendor specific things. So, it is good enough.
